datatable_tutorial/datatable_tutorial/datatable_tutorial.py

```diff
@@ -0,0 +1,269 @@
+import reflex as rx
+from typing import Any
+import asyncio
+import httpx
+
+
+class BaseState(rx.State):
+    pass
+
+   
+class DataTableState(BaseState):
+    """The app state."""
+
+    clicked_cell: str = "Cell clicked: "
+    edited_cell: str = "Cell edited: "
+    right_clicked_group_header : str = "Group header right clicked: "
+    item_hovered: str = "Item Hovered: "
+    deleted: str = "Deleted: "
+
+    cols: list[dict] = [
+        {"title": "Title", "type": "str"},
```

Comment:
what is this column for? currently it's showing a number that's the same as the row number

Reply:
that is all it is for currently, in case someone doesn't show the row number using the row_markers prop